agasc/supplement/magnitudes/mag_estimate.py

```diff
@@ -1365,17 +1365,18 @@ def get_agasc_id_stats(agasc_id, obs_status_override=None, tstop=None):
         )
         return result, stats, failures
 
-    if len(all_telem) - len(failures) <= 0:
+    # add failed observations to the list of excluded observations
+    excluded_obs += np.array([t is None for t in all_telem])
```

Comment:
I'm getting a little confused here, because I thought excluded_obs was a mask over the observations, and now this appends a new mask?  To me it seems that a separate mask would be helpful.

Reply:
Oh wait, you're not appending, you're just updating the mask in place by adding.  Would a logical operation be more readable?

Reply:
Yes, maybe I can use `|=`.

Note that this is a line that was just moved from below. It was there in the previous version, just after the check.

Reply:
For me `|=` would have helped.  I basically just read the comment and since the operation on "list" is often more of a append, that tripped me up.